src/wallet/wallet.cpp

```diff
@@ -1882,7 +1887,7 @@ CAmount CWallet::GetImmatureWatchOnlyBalance() const
 }
 
 void CWallet::AvailableCoins(vector<COutput> &vCoins,
-    bool fOnlyConfirmed,
+    bool onlyConfirmed, // Don't shadow the fOnlyConfirmed member variable
```

Comment:
nit: what about prepending `_`

Reply:
IDK, I never thought that the hungarian prefixes had much value anyway...

Reply:
I mean if we used the `_` we could left the `f` at the beginning which is our way of naming boolean.

Reply:
we have been using the _ notation in variables to differentiate between 2 variables in constructors that are meant to be exactly the same but can't be because one is a member and the other is the arg that the member variable is about to be set to.  

This is a different case.  the member variable fOnlyConfirmed never gets set to this argument, and controls spending.  But this arg controls just how this member function behaves.